src/chrome/content/rules/derstandard.at.xml

```diff
@@ -0,0 +1,49 @@
+<!--
+	Mismatch:
+		- meinparlament
+		- parship
+		- text
+		- tv
+
+	Problematic:
+		- redirect on some pages, only css, images and js are working
+-->
+<ruleset name="derStandard.at (partial)">
+	<target host="derstandard.at"/>
+	<target host="www.derstandard.at"/>
+	<target host="epaper.derstandard.at"/>
+	<target host="images.derstandard.at"/>
+	<target host="mobil.derstandard.at"/>
+
+	<exclusion pattern="^http://derstandard\.at/$" />
+	<exclusion pattern="^http://www\.derstandard\.at/$" />
+	<exclusion pattern="^http://mobil\.derstandard\.at/$" />
```

Comment:
I think those are useless since the following rules don't include those links anyway.